projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1580,11 +1580,17 @@ protected function get_related_post_ids( $post_id, $size, array $filters ) {
 			}
 		}
 
+		$user_agent = '';
+		if ( isset( $_SERVER['HTTP_USER_AGENT'] ) ) {
+			// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+			$user_agent = wp_unslash( $_SERVER['HTTP_USER_AGENT'] );
```

Comment:
You should be able to get rid of the PHPCS exception like so:

```suggestion
			$user_agent = filter_var( wp_unslash( $_SERVER['HTTP_USER_AGENT'] ) );
```

Maybe we should add a `strtolower` as well?

```suggestion
			$user_agent = strtolower( filter_var( wp_unslash( $_SERVER['HTTP_USER_AGENT'] ) ) );
```

Reply:
Sure, why not. I've made the change and added `strtolower`.